tests/test_with_oauth.rs

```diff
@@ -385,13 +385,15 @@ async fn test_playback() {
     // Restore the original playback data
     if let Some(backup) = &backup {
         let uri = backup.item.as_ref().map(|item| item.id());
-        let offset = None;
-        let device = backup.device.id.as_deref();
-        let position = backup.progress.map(|p| p.as_millis() as u32);
-        client
-            .start_uris_playback(uri, device, offset, position)
-            .await
-            .unwrap();
+        if let Some(uri) = uri {
```

Comment:
Is it necessary to extract the `uri` variable? It seems that `uri` will not be used later

Reply:
It's used in `start_uris_playback`